src/test/addresses_deployers.sol

```diff
@@ -31,7 +31,8 @@ contract Deployers {
     address public constant PE_04       = 0xdB33dFD3D61308C33C63209845DaD3e6bfb2c674;
     address public constant PE_05       = 0xDA01018eA05D98aBb66cb21a85d6019a311570eE;
     address public constant PE_06       = 0xDA0111100cb6080b43926253AB88bE719C60Be13;
-    address public constant CES         = 0x9956fca5a8994737f124c481cEDC6BB3dc5BF010;
+    address public constant CES_01      = 0x9956fca5a8994737f124c481cEDC6BB3dc5BF010;
+    address public constant CES_02      = 0xc0b362cbb0117Ec6A4b589f744d4dECb2768A2eB;
```

Comment:
the deployer `0xb27b6fa77d7fbf3c1bd34b0f7da59b39d3db0f7e` also looks to be missing in the list, as mentioned in previous comments, that was used for previous onboardings.
consider including it to improve coverage of `checkAuth` chainlog ward check tests.

Reply:
checking, previous rwa onboarding contracts were redeployed in a subsequent goerli spell with a deployer included in the list.
I'd double check if previous rwa onboarding deployers of contracts present in the chainlog were included and if not consider including them in a housekeeping pr eventually.